content/en/docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -6,7 +6,7 @@ weight: 30
 
 <!-- overview -->
 This page explains how to manage Kubernetes running on a specific
-cloud provider.
+cloud provider. There are many other third-party cloud provider projects, but this list is specific to projects embedded within, or relied upon by Kubernetes itself.
```

Comment:
At some point we should provide an overview of what cloud provider integration is (revising this page along with https://kubernetes.io/docs/concepts/architecture/cloud-controller/)

Reply:
Yeah, I agree! It'd be helpful.